microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -17,33 +17,22 @@ In this tutorial, you:
 *  Learn about WebView2 concepts along the way.
 
 The **Microsoft.UI.Xaml** (WinUI 2) package is part of the Windows UI Library.  This package provides Windows UI features, including:
-*  UWP XAML controls
-*  Dense control styles
+*  UWP XAML controls.
+*  Dense control styles.
 *  Fluent styles and materials.
 
 WinUI 2 supports UWP only.  These controls are backward-compatible.
 
 Follow the major Step sections in sequence, below.
 
-
-<!-- ====================================================================== -->
-## Step 1 - Download a preview channel of Microsoft Edge
-
-For full API compatibility of the **Microsoft.UI.Xaml** (WinUI 2) package and WebView2, this tutorial uses a _prerelease_ version of WebView2 SDK, together with a preview channel of Microsoft Edge.  (This tutorial doesn't use the other supported combination, which is a _release_ version of WebView2 SDK + the WebView2 Runtime.)
-
-1. If you haven't already, download any [Microsoft Edge Insider (preview) Channel](https://www.microsoftedgeinsider.com/download) (Beta, Dev, or Canary) on a supported operating system (OS):
-   *  Windows 7
-   *  Windows 8.1
-   *  Windows 10
-   *  Windows 11
-
-   We recommend using the Canary channel of Microsoft Edge.  The minimum required version is 82.0.488.0.
+<!-- skip h3 -->
```

Comment:
@jasonstephen15 - (entering a fresh Comment instead of Outdated).  I removed the section "Download a preview channel of Microsoft Edge" and ended up not replacing it by a section "Download the WebView2 Runtime".

Reply:
msg'd offline, so closing thread